data/starter_choose.s

```diff
@@ -0,0 +1,67 @@
+@ the fourth big chunk of data
```

Comment:
Why isn't all of this data properly extracted into the C file?

Reply:
Because I'm too lazy to dump graphics. :((